spec/configuration_spec.rb

```diff
@@ -24,4 +24,40 @@
 
     expect(SurveyGizmo::Connection.send(:connection).params).to eq('api_token' => 'slimthug', 'api_token_secret' => 'fourfourz')
   end
+
+  describe '#api=' do
+    it 'should set US region by default' do
+      SurveyGizmo.configure
+      expect(SurveyGizmo.configuration.api_url).to eq('https://restapi.surveygizmo.com')
+      expect(SurveyGizmo.configuration.api_time_zone).to eq('Eastern Time (US & Canada)')
+    end
+
+    it 'should set US region with :us symbol specified' do
+      SurveyGizmo.configure do |config|
+        config.region = :us
+      end
+
+      expect(SurveyGizmo.configuration.api_url).to eq('https://restapi.surveygizmo.com')
+      expect(SurveyGizmo.configuration.api_time_zone).to eq('Eastern Time (US & Canada)')
+    end
+
+    it 'should set EU region with :eu symbol specified' do
+      SurveyGizmo.configure do |config|
+        config.region = :eu
+      end
+
+      expect(SurveyGizmo.configuration.api_url).to eq('https://restapi.surveygizmo.eu')
+      expect(SurveyGizmo.configuration.api_time_zone).to eq('Berlin')
+    end
+
+    it 'should fail with an unavailable region' do
+      expect {
+        SurveyGizmo.configure do |config|
+          config.region = :cz
+        end
+      }.to raise_error
+    end
+
```

Comment:
trivial but the style of the project doesn't include trailing whitespace in class definitions